.ci/travis/travis.sh

```diff
@@ -288,8 +292,10 @@ no-exception-test-guava)
   mvn -e clean install -Pno-validations
   cd .ci-temp/contribution/checkstyle-tester
   export MAVEN_OPTS="-Xmx2048m"
-  groovy ./launch.groovy --listOfProjects projects-to-test-on.properties
+  groovy ./launch.groovy --listOfProjects projects-to-test-on.properties \
```

Comment:
please move this to separate commit, as it is not related to `.ci-temp` commit.

Reply:
Done.